contracts/crosschain/DuckiesAlienBridge.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.18;
+
+import '@openzeppelin/contracts/access/Ownable.sol';
+import '@layerzerolabs/solidity-examples/contracts/lzApp/NonblockingLzApp.sol';
+
+import '../YellowToken.sol';
+
+contract DuckiesAlienBridge is Ownable, NonblockingLzApp {
+	event BridgeOut(uint16 chainTo, address indexed sender, uint256 amount);
+	event BridgeIn(uint16 chainFrom, address indexed receiver, uint256 amount);
+
+	YellowToken duckiesContract;
+
+	constructor(address endpoint, address duckiesAddress) NonblockingLzApp(endpoint) {
+		duckiesContract = YellowToken(duckiesAddress);
+	}
+
+	function _nonblockingLzReceive(
+		uint16, // _srcChainId
+		bytes memory, // _srcAddress
+		uint64, // _nonce
+		bytes memory _payload
+	) internal override {
+		(address receiver, uint256 amount) = abi.decode(_payload, (address, uint256));
```

Comment:
Need very clear and specific validators here, also probably erecover if the message is signed or something

Reply:
What do we need validation for?
LZ handles all the validation